src/encoder/encodervorbis.cpp

```diff
@@ -37,39 +37,15 @@ A lot of stuff has been stolen from:
 
 EncoderVorbis::EncoderVorbis(EncoderCallback* pCallback)
         : m_bStreamInitialized(false),
+          m_vblock({}),
```

Comment:
> But since the current solution already initializes the members of question

I'm not sure what you mean. Currently, some members are *not* initialized. That's what this PR tries to address.

> In any case, we should not introduce a gcc 4.8 compiler warning.

Indeed. The suggestion would be to remove that particular warning from GCC since the GCC folks already recognized it was a mistake to make them so strict, and relaxed the warning for GCC 5.